### ISSUE TITLE
catching migration fails

### ISSUE THREAD

**Issue by toneplex** [tongueroo/ufo#99]
I've been playing with the `ufo task` with the wait flag to see if it will respond differently when the migration fails. From my tests, I don't see any difference. In a CI/CD setup, I need to know if the migrations fail and stop the deployment. Is there something I'm missing with regards to detecting a failed migration?

**Comment by tongueroo** [comment]
No, you're not missing anything. Unfortunately, the run task command doesn't seem to forward the exit signal from the migrate command that the ecs task runs. This is from the underlying ecs run-task api call. It only tells you that the container finished and stopped, vs not being able to start at all. So there's no info about the exit signal, at least the last time I checked - a few months ago. It could be different now. 🧐

For projects, I ended up setting codebuild where it calls `docker run` directly and does the migration.
